tests/python/unittest/test_tir_transform_split_host_device.py

```diff
@@ -168,5 +168,21 @@ def main_kernel(n: T.int32):
         return mod
 
 
+class TestSplitHostDevice(BaseCompare):
```

Comment:
Is this test relevant? Looks like a dup from https://github.com/apache/tvm/pull/14988

Reply:
Hmm, it was a dup from https://github.com/apache/tvm/pull/14982, which I had based this PR on top of to make allow the CI to use that bugfix.  I think the "squash and merge" should remove it from this PR as part of merging, but just to be on the safe side, rebasing onto main.

Reply:
And after rebase, no changes remain in `test_tir_transform_split_host_device.py`.